libs/core-ui/src/lib/util/JointDataset.ts

```diff
@@ -420,6 +420,23 @@ export class JointDataset {
     return [];
   }
 
+  public readonly getRawValue = (
+    v: number | undefined,
```

Comment:
nit: you could combine the two if statements to simplify the funciton: 
`if( v !== undefined &&  (meta.isCategorical || meta?.treatAsCategorical) && meta.sortedCategoricalValues))
 { return meta.sortedCategoricalValues[v]; }
return v;`

Reply:
Indeed! Not sure how I missed that 🙂 Thanks for pointing it out.